packages/formik/src/Formik.tsx

```diff
@@ -29,6 +29,7 @@ import {
   setIn,
   setNestedObjectValues,
 } from './utils';
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
Can you move this import up with the other lodash one and add a changeset please?

Reply:
Done